packages/trader/src/App/Components/Elements/ContractDrawer/__tests__/contract-drawer.spec.tsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import { createBrowserHistory } from 'history';
+import { render, screen } from '@testing-library/react';
+import { mockStore } from '@deriv/stores';
+import { Router } from 'react-router';
+import { isMobile, isDesktop } from '@deriv/shared';
+import ContractDrawer from '../contract-drawer';
+import TraderProviders from '../../../../../trader-providers';
+import userEvent from '@testing-library/user-event';
+
+const mocked_props = {
+    contract_info: '',
+    is_accumulator: false,
+    is_dark_theme: false,
+    is_market_closed: false,
+    is_multiplier: false,
+    is_vanilla: false,
+    is_smarttrader_contract: false,
+    is_sell_requested: false,
+    is_turbos: false,
+    onClickCancel: jest.fn(),
+    onClickSell: jest.fn(),
+    toggleHistoryTab: jest.fn(),
+} as unknown as React.ComponentProps<typeof ContractDrawer>;
```

Comment:
why are we making it unknown first?

Reply:
Because I didn't mocked all props, only those fields which are used directly in <ContractDrawerCard />